src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -255,16 +336,23 @@ def load_and_exec_node(
 
     @staticmethod
     def update_environment_variables_with_connections(connections: dict):
+        """Update environment variables with connections.
+
+        :param connections: A dictionary containing connection information.
+        :type connections: dict
+        :return: A dictionary containing updated environment variables.
+        :rtype: dict
+        """
         from promptflow._sdk._utils import update_environment_variables_with_connections
 
         return update_environment_variables_with_connections(connections)
 
-    def convert_flow_input_types(self, inputs: dict):
-        """
-        Convert flow inputs type if existing. Ignore missing inputs.
-
-        return:
-            type converted inputs
+    def convert_flow_input_types(self, inputs: dict) -> Mapping[str, Any]:
+        """Convert flow inputs type if existing. Ignore missing inputs.
+        :param inputs: _description_
```

Comment:
`_description_`?

Reply:
This PR just target to use GPT to generate something. (For the first stage)